relion/convert/convert_deprecated.py

```diff
@@ -709,30 +710,39 @@ def convertToMrc(fn):
     return fn
 
 
-def convertMask(img, outputPath, newDim=None):
-    """ Convert binary mask to a format read by Relion and truncate the
-    values between 0-1 values, due to Relion only support masks with this
-    values (0-1).
```

Comment:
I think this part is still valid, but we use relion_image_handler now to achieve the same

Reply:
I think you missed the same func below :)

Reply:
I was pointing to the part of the documentation saying that it will be truncated between 0 and 1, but it not a big deal anyway

Reply:
ah, sorry. Yes, the thresholding is done below as well , with image handler already